cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaConnectConfigurationBuilder.java

```diff
@@ -325,6 +399,39 @@ public KafkaConnectConfigurationBuilder withPluginPath() {
         return  this;
     }
 
+    /**
+     * Configures the Strimzi Metrics Reporter. It is set only if user enables Strimzi Metrics Reporter.
+     *
+     * @param model Strimzi Metrics Reporter configuration
+     *
+     * @return Returns the builder instance
+     */
+    public KafkaConnectConfigurationBuilder withStrimziMetricsReporter(MetricsModel model) {
+        printSectionHeader("Strimzi Metrics Reporter configuration");
+        if (model instanceof StrimziMetricsReporterModel reporterModel) {
+            writer.println(StrimziMetricsReporterConfig.LISTENER_ENABLE + "=true");
+            writer.println(StrimziMetricsReporterConfig.LISTENER + "=http://:" + MetricsModel.METRICS_PORT);
+            writer.println(StrimziMetricsReporterConfig.ALLOW_LIST + "=" + reporterModel.getAllowList());
+            writer.println("admin." + StrimziMetricsReporterConfig.LISTENER_ENABLE + "=true");
+            writer.println("admin." + StrimziMetricsReporterConfig.LISTENER + "=http://:" + MetricsModel.METRICS_PORT);
+            writer.println("admin." + StrimziMetricsReporterConfig.ALLOW_LIST + "=" + reporterModel.getAllowList());
+            writer.println("producer." + StrimziMetricsReporterConfig.LISTENER_ENABLE + "=true");
+            writer.println("producer." + StrimziMetricsReporterConfig.LISTENER + "=http://:" + MetricsModel.METRICS_PORT);
+            writer.println("producer." + StrimziMetricsReporterConfig.ALLOW_LIST + "=" + reporterModel.getAllowList());
+            writer.println("consumer." + StrimziMetricsReporterConfig.LISTENER_ENABLE + "=true");
+            writer.println("consumer." + StrimziMetricsReporterConfig.LISTENER + "=http://:" + MetricsModel.METRICS_PORT);
+            writer.println("consumer." + StrimziMetricsReporterConfig.ALLOW_LIST + "=" + reporterModel.getAllowList());
+        } else {
+            // we disable the listener just in case the user manually set SMR from spec.config
+            // because this would enable the listener on port 8080
+            writer.println(StrimziMetricsReporterConfig.LISTENER_ENABLE + "=false");
+            writer.println("admin." + StrimziMetricsReporterConfig.LISTENER_ENABLE + "=false");
+            writer.println("producer." + StrimziMetricsReporterConfig.LISTENER_ENABLE + "=false");
+            writer.println("consumer." + StrimziMetricsReporterConfig.LISTENER_ENABLE + "=false");
+        }
```

Comment:
Once again, I would leave this out. You do not know whether such a configuration is a mistake, or whether there is some intention behind it, etc.

Reply:
ok